client/me/purchases/payment-methods/main.tsx

```diff
@@ -24,7 +24,7 @@ function PaymentMethods(): JSX.Element {
 				brandFont
 				headerText={ titles.sectionTitle }
 				subHeaderText={ translate(
-					'Add or delete payment methods for your account. {{learnMoreLink}}Learn more{{/learnMoreLink}}.',
+					'Add, edit, or delete payment methods for your account. {{learnMoreLink}}Learn more{{/learnMoreLink}}.',
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.

Reply:
Updated to use existing translation.